src/sideof.jl

```diff
@@ -58,6 +58,7 @@ Possible results are `IN`, `OUT` or `ON` the `ring`.
 """
 function sideof(point::Point, ring::Ring)
   assertion(CoordRefSystems.ncoords(crs(point)) == 2, "points must have 2 coordinates")
+  point = Proj(crs(point))(point)
```

Comment:
```suggestion
  point = Proj(crs(ring))(point)
```
I believe we want to project to the crs of the other geometry?

Reply:
Yes, that was a mistake